tests/Fixer/Import/FullyQualifiedStrictTypesFixerTest.php

```diff
@@ -353,6 +353,31 @@ class SomeClass extends \Foo\Bar\A implements \Foo\Bar\Izumi, A, \A\B, \Foo\Bar\
             '<?php use A\B; class Foo { use \A\C; use \D; use \A\B { \A\B::bar as baz; } };',
         ];
 
+        yield 'typed property in function' => [
+            '<?php use A\B; function foo(B $v) {}',
+            '<?php use A\B; function foo(\A\B $v) {}',
+        ];
+
+        yield 'typed property in class' => [
+            '<?php use A\B; class Cl { public B $p; }',
+            '<?php use A\B; class Cl { public \A\B $p; }',
+        ];
+
+        yield 'typed promoted property in class' => [
+            '<?php use A\B; class Cl { public function __construct(B $p) {} }',
+            '<?php use A\B; class Cl { public function __construct(\A\B $p) {} }',
```

Comment:
```suggestion
            '<?php use A\B; class Cl { public function __construct(private B $p) {} }',
            '<?php use A\B; class Cl { public function __construct(private \A\B $p) {} }',
```

Reply:
👍 to test promoted property really, done